packages/ui/test/modals/AddMembershipModal.test.tsx

```diff
@@ -144,27 +145,51 @@ describe('UI: AddMembershipModal', () => {
     })
 
     describe('Success', () => {
+      const events = [
+        {
+          phase: { ApplyExtrinsic: 2 },
+          event: { index: '0x0502', data: [1] },
+        },
+        {
+          phase: { ApplyExtrinsic: 2 },
+          event: { index: '0x0000', data: [{ weight: 190949000, class: 'Normal', paysFee: 'Yes' }] },
+        },
+      ]
+
+      beforeEach(() => {
+        set(transaction, 'signAndSend', () => stubTransactionResult(events))
+      })
+
+      it('Renders transaction success', async () => {
+        const { getByText, findByText } = await renderAuthorizeStep()
+        fireEvent.click(getByText(/^sign and create a member$/i))
+
+        expect(await findByText('Success')).toBeDefined()
+        expect(getByText(/^realbobbybob/i)).toBeDefined()
+      })
+    })
+
+    describe('Failure', () => {
```

Comment:
This seems like a duplicate of the previous test and does not describe failure but success.

Reply:
Oh, my bad :( I'll fix that. Thanks.